spam/spam_tutorial.py

```diff
@@ -544,14 +681,14 @@ def plot_label_frequency(L):
 plot_label_frequency(L_train)
 
 # %% [markdown]
-# We see that over half of our training dataset data points have 0 or 1 weak labels.
-# Fortunately, the signal we do have can be used to train a classifier with a larger feature set than just these labeling functions that we've created, allowing it to generalize beyond what we've specified.
+# We see that over half of our `train` dataset data points have 2 or fewer labels from LFs.
+# Fortunately, the signal we do have can be used to train a classifier with a generic feature set, allowing it to generalize beyond what we've specified via our LFs.
```

Comment:
Nit: 'classifier with a generic feature set' still seems unclear. I think it's ok to just say the classifier runs on arbitrary text input (though that makes it somewhat specific to this example).